### ISSUE TITLE
Balances no longer shows N/A when logged out

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#6741]
It also doesn't show the LogIn button… 

Stoprint does, though. Interesting.

**Comment by drewvolz** [comment]
So here's what we want:
 - Balances should show N/A when logged out
 - The view should show the login button if logged out
 
 ---
 
 I've found a few pieces of documentation that could help us narrow what to change.
 
1. **For initial + pull-to-refresh**: getting `FormattedValueCell`'s loading to show the correct state (... ~> NA)
    > Use `isInitialLoading`. If you are using disabled or lazy queries, you can use the `isInitialLoading` flag instead. [It's a derived flag](https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading) that is computed from: `isLoading && isFetching` so it will only be true if the query is currently fetching for the first time.

2. **For initial load**: Showing the sections for an `error message` and the `login button` need more tweaking.
    - Note: pull to refresh ends up showing the correct state.


https://user-images.githubusercontent.com/5240843/214489590-32d85d66-4e10-4f7d-964e-0bdb1c52b57a.mp4


 
### When enabled is false

> - If the query has cached data, then the query will be initialized in the `status === 'success'` or `isSuccess` state.
> - If the query does not have cached data, then the query will start in the `status === 'loading'` and `fetchStatus === 'idle'` state.
> - The query will not automatically fetch on mount.
> - The query will not automatically refetch in the background.
> - The query will ignore query client `invalidateQueries` and `refetchQueries` calls that would normally result in the query refetching.
> - `refetch` returned from `useQuery` can be used to manually trigger the query to fetch.

From the example provided [in the documentation](https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isInitialLoading), we can expect a falsey enabled state to render like so:

1. data
1. isError
1. isLoading
1. isFetching

## Changes to isLoading and Idle state
[ Migrating to react query v4 docs](https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#disabled-queries) has a bit of rationale.

> #### The idle state has been removed
> With the introduction of the new [fetchStatus](https://tanstack.com/query/v4/docs/react/guides/queries#fetchstatus) for better offline support, the idle state became irrelevant, because fetchStatus: 'idle' captures the same state better. For more information, please read [Why two different states](https://tanstack.com/query/v4/docs/react/guides/queries#why-two-different-states).

```diff 
- status: 'idle'
+ status: 'loading'
+ fetchStatus: 'idle'
```

> Also, have a look at [the guide on dependent queries](https://tanstack.com/query/v4/docs/react/guides/dependent-queries)
> 
> This will mostly affect disabled queries that don't have any data yet, as those were in idle state before:

```diff
- isLoading
+ isInitialLoading
```

> Due to this change, disabled queries (even temporarily disabled ones) will start in `loading` state. To make migration easier, especially for having a good flag to know when to display a loading spinner, you can check for `isInitialLoading` instead of `isLoading`.
> 
> See also the guide on [disabling queries](https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isInitialLoading)

**Comment by drewvolz** [comment]
- Somewhat related is #6899

**Comment by drewvolz** [comment]
Closed by #6790
